### ISSUE TITLE
Why not publish submodule to the maven repository?

### ISSUE THREAD

**Issue by LeadroyaL** [secure-software-engineering/FlowDroid#149]
Hi, owner of FlowDroid.

I've use FlowDroid for long time. Recently, I just want to import soot-infolow(not the whole FlowDroid) into my project. I try to find it in mavenCentral and jCenter, neigher of  them hold FlowDroid module.

I always use gradle to manage my dependencies, I think it is necessary to publish every  submodule to the public repositores.

Would you mind take a thought to do the publishing? Thanks!

**Comment by StevenArzt** [comment]
FlowDroid is published to the artifactory at the university of Paderborn. There is one URL for snapshots and one for release builds:

```
	<repositories>
		<repository>
			<id>soot-snapshot</id>
			<name>Soot snapshot server</name>
			<url>https://soot-build.cs.uni-paderborn.de/nexus/repository/soot-snapshot/</url>
		</repository>
		<repository>
			<id>soot-release</id>
			<name>Soot release server</name>
			<url>https://soot-build.cs.uni-paderborn.de/nexus/repository/soot-release/</url>
		</repository>
	</repositories>
```

**Comment by LeadroyaL** [comment]
Yes, I know this repository. But I think mavenCentral is more popular and developer can get FlowDroid in a convenient way.

**Comment by LeadroyaL** [comment]
I find the guide [https://central.sonatype.org/pages/ossrh-guide.html](https://central.sonatype.org/pages/ossrh-guide.html) . It need register an account and config in pom.xml, I hope you can upload the release not only in university of Paderborn but also in mavenCentral. Thanks.

**Comment by StevenArzt** [comment]
We're working on releasing the artifacts to Maven Central.

**Comment by LeadroyaL** [comment]
@StevenArzt Hi, I find only one module at https://mvnrepository.com/artifact/de.tud.sse/soot-infoflow .
Could you please upload [soot-infoflow-android] to mavenCentral together?

**Comment by XhstormR** [comment]
Yes, the public maven repository only contains soot-infoflow, and there is no soot-Infoflow-Android.

https://repo1.maven.org/maven2/de/tud/sse/
